lib/oauth2-helper.js

```diff
@@ -132,8 +133,8 @@ function validateClient(client, options, next) {
     var authorizedScopes = normalizeList(client.scopes);
     var requestedScopes = normalizeList(options.scope);
     if (authorizedScopes.length && !isScopeAuthorized(requestedScopes, authorizedScopes)) {
-      err = new AuthorizationError(
-          'Unauthorized scope: ' + options.scope, 'access_denied');
+      err = new AuthorizationError(g.f(
+          'Unauthorized scope: %s', options.scope), 'access_denied');
```

Comment:
same .. I disagree .. it is within a normal phrase and could be translated. It is neither an argument or variable so I see no harm if translated